packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -1037,15 +1037,14 @@ export class AvailableSlotsService {
       logger.settings.minLevel = 2;
     }
 
+    const disableRollingWindowAdjustment = input.disableRollingWindowAdjustment;
     const isRollingWindowPeriodType = eventType.periodType === PeriodType.ROLLING_WINDOW;
     const startTimeAsIsoString = input.startTime;
     const isStartTimeInPast = dayjs(startTimeAsIsoString).isBefore(dayjs().subtract(1, "day").startOf("day"));
 
-    // If startTime is already sent in the past, we don't need to adjust it.
-    // We assume that the client is already sending startTime as per their requirement.
-    // Note: We could optimize it further to go back 1 month in past only for the 2nd month because that is what we are putting a hard limit at.
```

Comment:
Why did we delete this comment??? Please leave as is and in this case there is no need for a new comment because constant disableRollingWindowAdjustment is self explanatory.